### DISCUSSION TITLE
onthisday/10/10-03/

### DISCUSSION THREAD

**Discussion by giscus** [FanzOfLenaZavaroni/fanzoflenazavaroni.github.io#50] [General]
# onthisday/10/10-03/

Details for On This Day are genarated by the posts added to the website so the content is subject to changes/updates over time.

https://fanzoflenazavaroni.github.io/onthisday/10/10-03/

**Comment by StephenMolloy1** [comment]
On the British Newspaper archive, todays edition of Wales on Sunday (page 8) gives Lena's time of death as 6:30 pm.
https://www.britishnewspaperarchive.co.uk/titles/wales-on-sunday

**Comment by StephenMolloy1** [comment]
Today's 1999 edition of course, though I'm sure you guessed this.

**Comment by FanzOfLenaZavaroni** [comment]
Thanks for the details and link.
